node/_events.d.ts

```diff
@@ -0,0 +1,788 @@
+   *     // processes one event at a time (even with await). Do not use
+   *     // if concurrent execution is required.
+   *     console.log(event); // prints ['bar'] [42]
+   *   }
+   *   // Unreachable here
+   * })();
+   *
+   * process.nextTick(() => ac.abort());
+   * ```
+   * @since v13.6.0, v12.16.0
+   * @param eventName The name of the event being listened for
+   * @return that iterates `eventName` events emitted by the `emitter`
+   */
+  static on: typeof on;
+  /**
+   * A class method that returns the number of listeners for the given `eventName`registered on the given `emitter`.
+   *
+   * ```js
+   * const { EventEmitter, listenerCount } = require('events');
+   * const myEmitter = new EventEmitter();
+   * myEmitter.on('event', () => {});
+   * myEmitter.on('event', () => {});
+   * console.log(listenerCount(myEmitter, 'event'));
+   * // Prints: 2
+   * ```
+   * @since v0.9.12
+   * @deprecated Since v3.2.0 - Use `listenerCount` instead.
+   * @param emitter The emitter to query
+   * @param eventName The event name
+   */
+  static listenerCount(
+    emitter: EventEmitter,
+    eventName: string | symbol,
+  ): number;
+
+  /**
+   * Returns a copy of the array of listeners for the event named `eventName`.
+   *
+   * For `EventEmitter`s this behaves exactly the same as calling `.listeners` on
+   * the emitter.
+   *
+   * For `EventTarget`s this is the only way to get the event listeners for the
+   * event target. This is useful for debugging and diagnostic purposes.
+   *
+   * ```js
+   * const { getEventListeners, EventEmitter } = require('events');
+   *
+   * {
+   *   const ee = new EventEmitter();
+   *   const listener = () => console.log('Events are fun');
+   *   ee.on('foo', listener);
+   *   getEventListeners(ee, 'foo'); // [listener]
+   * }
+   * {
+   *   const et = new EventTarget();
+   *   const listener = () => console.log('Events are fun');
+   *   et.addEventListener('foo', listener);
+   *   getEventListeners(et, 'foo'); // [listener]
+   * }
+   * ```
+   * @since v15.2.0
+   */
+  static getEventListeners: typeof getEventListeners;
+
+  /**
+   * This symbol shall be used to install a listener for only monitoring `'error'`
+   * events. Listeners installed using this symbol are called before the regular
+   * `'error'` listeners are called.
+   *
+   * Installing a listener using this symbol does not change the behavior once an
+   * `'error'` event is emitted, therefore the process will still crash if no
+   * regular `'error'` listener is installed.
+   */
+  static readonly errorMonitor: unique symbol;
+  static readonly captureRejectionSymbol: unique symbol;
+  /**
+   * Sets or gets the default captureRejection value for all emitters.
+   */
+  // TODO: These should be described using static getter/setter pairs:
+  static captureRejections: boolean;
+  static defaultMaxListeners: number;
+}
+
+export default EventEmitter;
```

Comment:
The static properties and the named exports for EventEmitter are the same functions, however there is no clean way to represent such a thing in TypeScript without doing duplicate declarations